charts/plgd-hub/templates/http-gateway/api-ingress.yaml

```diff
@@ -5,44 +5,31 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: {{ $fullname }}
+  name: {{ $fullname }}-api
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "plgd-hub.labels" . | nindent 4 }}
   annotations:
-{{/*    kubernetes.io/ingress.class: "nginx"*/}}
-    nginx.org/grpc-services: {{ $fullname | quote }}
-    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
-    ingress.kubernetes.io/force-ssl-redirect: "true"
-    nginx.ingress.kubernetes.io/enable-cors: "true"
-  {{- with .Values.httpgateway.ingress.annotations }}
-    {{- toYaml . | nindent 4 }}
-  {{- end }}
+    {{- if .Values.httpgateway.ingress.api.annotations }}
+    {{- include "plgd-hub.tplvalues.render" ( dict "value" .Values.httpgateway.ingress.api.annotations "context" $ ) | nindent 4 }}
+    {{- end }}
+    {{- if .Values.httpgateway.ingress.api.customAnnotations }}
+    {{- include "plgd-hub.tplvalues.render" ( dict "value" .Values.httpgateway.ingress.api.customAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
 spec:
   tls:
     - hosts:
         - {{ include "plgd-hub.httpgateway.apiDomain" . | quote }}
-        - {{ include "plgd-hub.httpgateway.uiDomain" . | quote }}
       {{- if $.Values.global.enableWildCartCert }}
       secretName: {{ include "plgd-hub.wildCardCertName" . | quote }}
       {{- else }}
       secretName: {{ include "plgd-hub.httpgateway.domainCertName" . | quote }}
       {{- end }}
   rules:
-  - host: {{ include "plgd-hub.httpgateway.uiDomain" . | quote }}
-    http:
-      paths:
-      - path: /
-        pathType: Prefix
-        backend:
-          service:
-            name: {{ $fullname }}
-            port:
-              number: {{ $port }}
   - host: {{ include "plgd-hub.httpgateway.apiDomain" . | quote }}
     http:
       paths:
-      {{- range  .Values.httpgateway.ingress.paths }}
+      {{- range  .Values.httpgateway.ingress.api.paths }}
```

Comment:
Some customers are hosting API paths on the root domain and skip adding `api.` prefix by setting `grpcgateway.domain`, `httpgateway.apidomain` and `certificateauthority.domain` on their own to overwrite this automated prefix. Are you sure this won't be broken? Can you test such a deployment, where you set these properties to same value as global.domain and verify everything is accessible on the root domain?

Reply:
There is only change that value for `httpgateway.ingress.paths` was setup to `httpgateway.ingress.api.paths`. The same for ui part. Logic how ingress handle Path is still the same